src/models/system.jl

```diff
@@ -21,6 +21,7 @@ function system!(out::Vector{T}, dx, x, sys, t) where {T <: Real}
     V_r = @view x[1:bus_size]
     V_i = @view x[(bus_size + 1):bus_vars_count]
     Sbase = PSY.get_basepower(sys)
+    # TODO: Don't create this matrices here every iteration
```

Comment:
Are we planning to move I_injections_r update in place? We still need to reset them at the beginning if we are adding current per device.

Reply:
We probably want to use StaticArrays or something like that